examples/computed-field-key/computed_field_key.md

```diff
@@ -0,0 +1,8 @@
+# Computed field key
+
+Ballerina allows you to have computed values as keys in a map. This can be done by specifying the key as an expression within square brackets.
+This is particularly useful when you want to define and use constants for key values.
```

Comment:
I think this second sentence reduce the value of the feature or it does not convey your idea yet.

Reply:
Any particular reason? 

This is from the [slides](https://ballerina.io/learn/slides/language-walkthrough/Ballerina_Language_Presentation-2021-03-08.pdf). This is a commonly asked question though. People usually use constants directly as keys and wonder why it doesn't work in the way they expect.

Reply:
okay. I think we give too much emphasis to that here that is all.  What if we remove the word `particularly`?

Reply:
Does it make a difference in the emphasis though? How about 

```suggestion
This feature can be used when you want to define and use constants for key values.
```

Reply:
Yes, this is good. Maybe this is just a subjective comment. I am okay to keep it as it is or for the new one.

Reply:
Will keep as is then. Thanks!